independent-publisher-connectors/Star Wars/apiDefinition.swagger.json

```diff
@@ -0,0 +1,862 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Star Wars",
+		"description": "The Star Wars API, is the world's first quantified and programmatically-accessible data source for all the data from the Star Wars canon universe! We've taken all the rich contextual stuff from the universe and formatted into something easier to consume with software.",
+		"version": "1.0",
+		"contact": {
+			"name": "Paul Culmsee",
+			"url": "https://www.sevensigma.com.au",
+			"email": "paul.culmsee@sevensigma.com.au"
+		}
+	},
+	"host": "swapi.dev",
+	"basePath": "/api",
+	"schemes": [
+		"https"
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/species": {
+			"get": {
+				"operationId": "GetSpecies",
+				"description": "A Species resource is a type of person or character within the Star Wars Universe",
+				"responses": {
+					"200": {
+						"description": "An array of species.",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"count": {
+									"type": "integer",
+									"format": "int32",
+									"description": "Total number of Species",
+									"title": "Species Count"

```

Comment:
@paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
@sriyen-msft apologies (I am slowly getting the hang of it)